conf/machine/imx8mm-lpddr4-evk.conf

```diff
@@ -39,3 +39,8 @@ DDR_FIRMWARE_NAME = " \
 "
 
 IMXBOOT_TARGETS_BASENAME = "flash_evk"
+
+MFGTOOLCONFIG = "bootloader.uuu.in fullimage.uuu.in"
+MFGTOOLCONFIG[bootloader.uuu.in] = "imx-boot:do_deploy ,imx-boot"
+MFGTOOLCONFIG[fullimage.uuu.in] = "imx-boot:do_deploy ${IMAGE_BASENAME}:do_image_complete ,imx-boot ${IMAGE_BASENAME}-${MACHINE}.wic"
```

Comment:
The space before the comma on these two lines looks odd. Did you have a problem without it? If so, then probably something to look into. If not, I suggest dropping it for consistency.